### ISSUE TITLE
NetworkImage provider issue

### ISSUE THREAD

**Issue by dchrzanowski** [bluefireteam/photo_view#473]
Flutter version 2.5.1, dart version 2.14.12, photo_view version 0.12.0

Library used to work fantastically up until recently.

I just received a new requirements in work. Had to update flutter and any other deps, since the build failed.

What I get now with with NetworkImage provider is a Red box with a cross inside and that's it, sometimes a broken image icon. The AssetImage provider works fine.
I thought that perhaps I'm having network issues, but the Image.network() loads up the image without any issues.

Maybe an issue with the server to app communication, but a basic image from https://via.placeholder.com/300.png/09f/fff fails to load with photo_view.

Kind Regards

EDIT:

My current workaround is to use PhotoView.customChild:
```dart
PhotoView.customChild(
  child: Image.network(
    widget.file.url!,
    fit: BoxFit.cover,
    height: MediaQuery.of(context).size.height,
    width: MediaQuery.of(context).size.width,
    alignment: Alignment.center,
  ),
),
```

**Comment by renancaraujo** [comment]
Thanks for reporting. Dupe of #464 in which the fix is now published on 0.13.0

**Comment by dchrzanowski** [comment]
@renancaraujo Thank you for fixing! Fantastic news!
